### DISCUSSION TITLE
Smart Synchronization of NAS to two External SSDs

### DISCUSSION THREAD

**Discussion by ChristopherWSmith** [iterate-ch/cyberduck#16084] [Q&A]
I've started to setup Mountain Duck - my intent is to utilize Mountain Duck to backup my NAS to multiple SSDs (NAS is too large for a single SSD). 

From what I can tell, it is the Cache in Preferences that determines where the offline files are stored, so I can change the cache to store in my SSD.  Is it possible to setup a separate cache per connection?

Also, is it possible to have two open connections to a single location?

I'm not sure if I'm headed down the right path, this is just what first made sense with the goal in mind.

**Comment by tatze96** [comment]
Correct. The cache location determines where the synchronization cache gets written. However, as the cache is obfuscated, you can only access the files from within the mounted drive. 

Bookmarks can generally also be mounted while using the _Smart Synchronization_ mode if the reachability to the server isn't available. Note that you'll only see the items already located within the cache either as metadata or as fully downloaded files if the synchronization gets paused and you are, therefore, working offline.

> Is it possible to setup a separate cache per connection?

There is currently no way to configure a separate cache location per connection.

> Also, is it possible to have two open connections to a single location?

I'm not sure if I understand the question correctly. You can create multiple bookmarks for the same server, for example, to mount specific folders separately.
